src/components/navbar/Navbar.jsx

```diff
@@ -1,9 +1,16 @@
+import React from 'react';
 import { NavLink } from 'react-router-dom';
+import { useWishlist } from '../../context/wishlist-context';
+import { useCart } from '../../context/cart-context';
 import { SearchBar } from '../component_index';
```

Comment:
suggestion: create index.js in context and export all context from it. so you can import all contexts with the same path.